base/hashing.jl

```diff
@@ -344,6 +344,293 @@ load_le(::Type{T}, ptr::Ptr{UInt8}, i) where {T <: Union{UInt32, UInt64}} =
     return hash_mix(a ⊻ secret[4], b ⊻ secret[2] ⊻ i)
 end
 
+@inline function load_le_array(::Type{UInt64}, arr::AbstractArray{UInt8}, idx)
```

Comment:
These look like very regular byte swapping functions to me. I don't think we have these, surprisingly, but as they are more generally useful (I've written similar code lots of times in packages), I wonder whether it makes sense to make them explicit with proper names.

Reply:
Then what is it? Rapidhash originally does endian conversion through byteswaps, if necessary on the platform: https://github.com/Nicoshev/rapidhash/blob/0d48fd50149dbef269ed97d63f24d14c14b57973/rapidhash.h#L199